src/schema/alerts.ts

```diff
@@ -140,10 +147,22 @@ export const updateAlerts = async (
   }
 
   if (!alerts) {
-    return saveReturnAlerts(await repo.save({ userId, ...data }));
+    return saveReturnAlerts(
```

Comment:
General question, why do we keep saving the full object instead of updating only the relevant field?

Reply:
Data is only one part of the alerts, not the whole object.
Same with flags, it's one or multiple flags.